dot/parachain/network-bridge/receiver.go

```diff
@@ -411,6 +413,8 @@ func getTopologyPeers(authorityDiscoveryService AuthorityDiscoveryService,
 
 type AuthorityDiscoveryService interface {
 	GetPeerIDByAuthorityID(authorityID parachaintypes.AuthorityDiscoveryID) peer.ID
+	GetAddressesByAuthorityID(authority parachaintypes.AuthorityDiscoveryID) *map[multiaddr.Multiaddr]struct{}
+	GetAuthorityIDsByPeerID(peerID peer.ID) *map[parachaintypes.AuthorityDiscoveryID]struct{}
```

Comment:
In Go, it's not required to return a pointer to a map because maps in Go are reference types. That means when you pass a map to a function or return it from a function, you are passing a reference to the underlying data, not a full copy of the map.

```suggestion
	GetAddressesByAuthorityID(authority parachaintypes.AuthorityDiscoveryID) map[multiaddr.Multiaddr]struct{}
	GetAuthorityIDsByPeerID(peerID peer.ID) map[parachaintypes.AuthorityDiscoveryID]struct{}
```